documentation/modules/ROOT/pages/transformations/outbox-event-router.adoc

```diff
@@ -205,14 +205,77 @@ You can use one of the following methods to configure the connector to apply the
+== Payload serialization format
+
+The outbox event router SMT supports arbitrary payload formats. 
+The SMT passes on `payload` column values that it reads from the outbox table without modification. 
+The way that the SMT converts these column values into Kafka message fields depends on how you configure the SMT.
+Common payload formats for serializing data are JSON and Avro.
+
+
+// Type: concept
+// Title: Using JSON as the serialization format
+// ModuleID: outbox-event-router-using-json-payload-format
+[[using-json-payload-format]]
+=== Using JSON as the payload format
+
+The default serialization format for the outbox event router SMT is JSON. 
+To use this format, the data type of the source column must be JSON (for example, `jsonb` in PostgreSQL).
+
+// Type: concept
+// Title: Producing Expanding escaped JSON String as JSON
+// ModuleID: outbox-event-router-expanding-escaped-json-string-as-json
+[[expanding-escaped-json-string-as-json]]
+==== Expanding escaped JSON String as JSON
+
+When a {prodname} outbox message represents the `payload` as a JSON String, the resulting Kafka message escapes the string as in the following example:
+
+[source,javascript,indent=0]
+----
+# Kafka Topic: outbox.event.order
+# Kafka Message key: "1"
+# Kafka Message Headers: "id=4d47e190-0402-4048-bc2c-89dd54343cdc"
+# Kafka Message Timestamp: 1556890294484
+{
+  "{\"id\": 1, \"lineItems\": [{\"id\": 1, \"item\": \"Debezium in Action\", \"status\": \"ENTERED\", \"quantity\": 2, \"totalPrice\": 39.98}, {\"id\": 2, \"item\": \"Debezium for Dummies\", \"status\": \"ENTERED\", \"quantity\": 1, \"totalPrice\": 29.99}], \"orderDate\": \"2019-01-31T12:13:01\", \"customerId\": 123}"
+}
+----
+
+The outbox event router enables you to expand the message content to "real" JSON, deducing the companion schema from the JSON document. 
+The resulting Kafka message is formatted as in the following example:
+
+[source,javascript,indent=0]
+----
+# Kafka Topic: outbox.event.order
+# Kafka Message key: "1"
+# Kafka Message Headers: "id=4d47e190-0402-4048-bc2c-89dd54343cdc"
+# Kafka Message Timestamp: 1556890294484
+{
+  "id": 1, "lineItems": [{"id": 1, "item": "Debezium in Action", "status": "ENTERED", "quantity": 2, "totalPrice": 39.98}, {"id": 2, "item": "Debezium for Dummies", "status": "ENTERED", "quantity": 1, "totalPrice": 29.99}], "orderDate": "2019-01-31T12:13:01", "customerId": 123
+}
+----
+
+To enable use of the outbox event router transformation, set the xref:outbox-event-router-property-table-expand-json-payload[`table.expand.json.payload`] to true, and use the `JsonConverter` as shown in the following example:
+
+[source]
+----
+transforms=outbox,...
+transforms.outbox.type=io.debezium.transforms.outbox.EventRouter
+transforms.outbox.table.expand.json.payload=true
+value.converter=org.apache.kafka.connect.json.JsonConverter
+----
+
 // Type: concept
 // Title: Using Avro as the payload format in {prodname} outbox messages
```

Comment:
```suggestion
// Title: Using Apache Avro as the payload format in {prodname} outbox messages
```

Reply:
I know that this is pre-existing content, and not directly related to this PR, but I'd appreciate it if you could correct this oversight as part of your update. 🙏🏼